bashbrew/go/src/bashbrew/cmd-push.go

```diff
@@ -44,7 +45,7 @@ func cmdPush(c *cli.Context) error {
 
 				created := dockerCreated(tag)
 				lastUpdated := fetchDockerHubTagMeta(tag).lastUpdatedTime()
-				if created.After(lastUpdated) {
+				if created.After(lastUpdated) || withForce {
```

Comment:
I think we should probably skip `fetchDockerHubTagMeta` in the case of `--force`, something like:
```go
if !withForce {
	created := dockerCreated(tag)
	lastUpdated := fetchDockerHubTagMeta(tag).lastUpdatedTime()
	if !created.After(lastUpdated) {
		fmt.Fprintf(os.Stderr, "skipping %s (created %s, last updated %s)\n", tag, created.Local().Format(time.RFC3339), lastUpdated.Local().Format(time.RFC3339))
		continue
	}
}
fmt.Printf("Pushing %s\n", tag)
...
```